packages/markup-frontstage/README.md

```diff
@@ -0,0 +1,52 @@
+# @bentley\markup-frontstage-react
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+The markup-frontstage-react provides Markup Frontstage for adding, editing, deleting markup elements in iModel.js application.
+
+## MarkupFrontstageProvider
+
+FrontstageProvider object that requires ViewState, IModelConnection, Editable boolean flag whether or not the markup elements are editable, ConfigurableUiControlContructor, Svg string as optional, ViewElementDictionary with 9 zone layouts to be added in the frontstage as optional, EmphasizeElementsProps as optional to instantiate. It's onAddMarkupEvent allows to raise event on adding markup, onstopMarkupEvent allows to raise event on stopping markup. Additionally there's onCloseAsync and onSaveAsync properties to be called to close the Frontstage and Save the Markup data respectively.
+
+## MarkupSettingsPanel
+
+It is the flyover component used to set colors, width, opacity of the markup element by MarkupFronstage.
```

Comment:
misspelling here, MarkupFrontstage

Reply:
Added space in between the words!

Reply:
I thought it was just the space between, now I see Frontstage spelling was wrong, corrected please :)